insights-ansible-playbook-verifier.spec

```
@@ -28,13 +28,15 @@ playbooks generated by Red Hat Insights.
 %pyproject_install insights_ansible_playbook_lib insights_ansible_playbook_verifier
 mkdir -p %{buildroot}/%{_libexecdir}
 mv %{buildroot}%{_bindir}/%{name} %{buildroot}%{_libexecdir}
+mkdir -p %{buildroot}/%{_sharedstatedir}/%{name}/
```

```
$ grep -rn _sharedstatedir /usr/lib/rpm/macros*
%_sharedstatedir        %{_prefix}/com
```
definitely not the right macro to use...

rather use:
```suggestion
mkdir -p %{buildroot}%{_localstatedir}/lib/%{name}
```

---

Interesting, https://docs.fedoraproject.org/en-US/packaging-guidelines/RPMMacros/ defines it as /var/lib.